src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -23,21 +23,54 @@
 
     <section name="What is Javadoc comment">
       <p>
-      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition or field definition.
+      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition, field or annotation field definition.
+      If a number of comments are put sequentially, only the last one will be used. The first sentence has a special meaning and should be clear and punchy, short and ended by a dot symbol.
```

Comment:
>  only the last one will be used

we need to change wording here, as in example below last comment with javadoc identifier is `/** Not a javadoc (ignored). */`
We need to define order/direction for "last" - from bottom to top.